### PR TITLE
[Three] Added missing methods in Transform Controls

### PR DESCRIPTION
**Pull request by stefan-lacatus**
Please fill in this template.

- [X ] Use a meaningful title for the pull request. Include the name of the package modified.
- [X ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/mrdoob/three.js/blob/dev/examples/js/controls/TransformControls.js

### CODE REVIEW

File: types/three/three-core.d.ts [hunk 1]
```diff
@@ -5110,6 +5110,7 @@ export class Mesh extends Object3D {
     drawMode: TrianglesDrawModes;
     morphTargetInfluences?: number[];
     morphTargetDictionary?: { [key: string]: number; };
+	isMesh: boolean;
```
Comment (dhritzkiv) [inline types/three/three-core.d.ts]: Was this meant to be part of the PR? It might be better to add it in a separate PR

**Review by dhritzkiv — APPROVED** [review]

File: types/three/three-core.d.ts (hunk 1 above)
Comment (efokschaner) [reply types/three/three-core.d.ts]: Also note that the indentation is off here.
